phpBB/styles/subsilver2/template/mcp_forum.html

```diff
@@ -57,24 +57,32 @@
 <!-- END topicrow -->
 <!-- IF not S_MERGE_SELECT -->
 <tr>
-	<td class="cat" colspan="6" align="{S_CONTENT_FLOW_END}">
-		<select name="action">
-			<option value="" selected="selected">{L_SELECT_ACTION}</option>
-			<!-- IF S_CAN_DELETE --><option value="delete_topic">{L_DELETE}</option><!-- ENDIF -->
-			<!-- IF S_CAN_RESTORE --><option value="restore_topic">{L_RESTORE}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MERGE --><option value="merge_topics">{L_MERGE}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MOVE --><option value="move">{L_MOVE}</option><!-- ENDIF -->
-			<!-- IF S_CAN_FORK --><option value="fork">{L_FORK}</option><!-- ENDIF -->
-			<!-- IF S_CAN_LOCK --><option value="lock">{L_LOCK}</option><option value="unlock">{L_UNLOCK}</option><!-- ENDIF -->
-			<!-- IF S_CAN_SYNC --><option value="resync">{L_RESYNC}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MAKE_NORMAL --><option value="make_normal">{L_MAKE_NORMAL}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MAKE_STICKY --><option value="make_sticky">{L_MAKE_STICKY}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MAKE_ANNOUNCE -->
-				<option value="make_announce">{L_MAKE_ANNOUNCE}</option>
-				<option value="make_global">{L_MAKE_GLOBAL}</option>
-			<!-- ENDIF -->
-		</select>
-		<input class="btnmain" type="submit" value="{L_SUBMIT}" />
+	<td class="cat" colspan="6">
+		<table width="100%" cellspacing="0">
```

Comment:
Why did you add a table here? You shouldn't modify the markup just because it might be more convenient for your expected usage.